### DISCUSSION TITLE
Subscriptions/Websockets metrics

### DISCUSSION THREAD

**Discussion by nadineoliveiramindera** [Netflix/dgs-framework#1622] [Q&A]
Before migrate to dgs framework we were using graphql kickstart, and we were getting some metrics from graphql.kickstart.spring.web.boot.metrics.WebsocketMetrics like sessions count (graphql.websocket.sessions) and subscriptions count (graphql.websocket.subscriptions). 
Now we want to have the same metrics using dgs framework, but we are not able to find any class that expose it. Do you know any way to do it?

**Comment by hzariv** [comment]
See https://netflix.github.io/dgs/advanced/instrumentation/#metrics-out-of-the-box
